llmfoundry/command_utils/eval.py

```diff
@@ -82,7 +82,7 @@ def evaluate_model(
         warnings.warn(
             VersionedDeprecationWarning(
                 'The argument fsdp_config is deprecated. Please use parallelism_config instead.',
-                remove_version='0.13.0',
+                remove_version='0.14.0',
```

Comment:
we're punting this deprecation until later?

Reply:
see pr description. its just for this function, and it was added with only version of deprecation when it should have been two